### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by veggiemonk**
Hi awesome-docker peeps!

We need a code of conduct for the community.

### CODE REVIEW

**Comment by devhkr** [comment]
👍 @veggiemonk  

A reference with a link from `CONTRIBUTING.md` to this document would be nice

**Comment by devhkr** [comment]
I'll fix the Travis build error

**Comment by veggiemonk** [comment]
@devhkr url changed

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 We appreciate and recognize [all contributors](https://github.com/veggiemonk/awesome-docker/graphs/contributors).
 
-Please note that this project is released with a [Contributor Code of Conduct](https://github.com/sindresorhus/awesome/blob/master/code-of-conduct.md). By participating in this project you agree to abide by its terms.
+Please note that this project is released with a [Contributor Code of Conduct](https://github.com/veggiemonk/awesome-docker/blob/master/CODE-OF-CONDUCT.md). By participating in this project you agree to abide by its terms.
```
Comment (devhkr) [inline CONTRIBUTING.md]: File name is `CODE_OF_CONDUCT.md`

**Comment by veggiemonk** [comment]
@devhkr thanks

**Comment by devhkr** [comment]
👍 @veggiemonk

**Comment by vegasbrianc** [comment]
@veggiemonk Great idea!

**Comment by vegasbrianc** [comment]
@veggiemonk Great idea!

**Pull request merged**